code/modules/mob/living/simple_animal/hostile/moghes.dm

```diff
@@ -0,0 +1,153 @@
+//animals from moghes
+/mob/living/simple_animal/hostile/biglizard
+	name = "plains tyrant"
+	desc = "The Skrazi, or 'plains tyrant', is an apex predator from the world of Moghes. Reaching up to fifteen feet in height, these beasts have earned a near-mythical reputation among the Unathi, with hunting a tyrant having been historically considered an elaborate form of suicide. Since the Contact War, these mighty predators are increasingly endangered due to loss of their habitats."
+	icon = 'icons/mob/biglizard.dmi'
+	icon_state = "biglizard"
+	icon_living = "biglizard"
+	icon_dead = "biglizard_dead"
+	speak_emote = list("hisses","roars")
+	emote_hear = list("hisses","grumbles","growls")
+	emote_see = list("hisses ferociously", "stomps")
+	emote_sounds = list('sound/effects/creatures/monstergrowl.ogg')
+	turns_per_move = 5
+	speak_chance = 5
+	meat_type = /obj/item/reagent_containers/food/snacks/meat
+	meat_amount = 10
+	organ_names = list("chest", "lower body", "left arm", "right arm", "left leg", "right leg", "head")
+	response_help  = "pets"
+	response_disarm = "shoves"
+	response_harm   = "harmlessly punches"
+	blood_overlay_icon = null
+	maxHealth = 500
+	health = 500
+	harm_intent_damage = 0
+	melee_damage_lower = 40
+	melee_damage_upper = 40
+	armor_penetration = 20
+	resist_mod = 10
+	mob_size = 30
+	environment_smash = 2
+	attacktext = "chomped"
+	attack_sound = 'sound/weapons/bloodyslice.ogg'
+
+	faction = "lizard"
+	butchering_products = list(/obj/item/stack/material/animalhide/lizard = 20)
+	var/is_devouring = FALSE
+
+/mob/living/simple_animal/hostile/biglizard/AttackingTarget()
+	. =..()
+	var/mob/living/L = .
+	if(istype(L))
+		if(prob(25))
+			L.Weaken(3)
+			L.visible_message("<span class='danger'>\The [src] knocks down \the [L]!</span>")
+
+/mob/living/simple_animal/hostile/biglizard/death(gibbed)
+	..()
+	anchored = TRUE
+
+/mob/living/simple_animal/hostile/biglizard/Life()
+	..()
+	adjustBruteLoss(-5)
+
+/mob/living/simple_animal/hostile/biglizard/verb/devour(mob/living/target as mob in oview())
+	set category = "Plains Tyrant"
+	set name = "Devour"
+	set desc = "Devours a creature, destroying its body and regenerating health."
+
+	if(!Adjacent(target))
+		return
+
+	if(target.isSynthetic())
```

Comment:
Imho it should still be able to devour synths, it just shouldn't gain any health from it and the message should be updated to indicate that it ripped parts form it instead of "flesh".